op-monitorism/multisig/monitor.go

```diff
@@ -122,7 +129,7 @@ func (m *Monitor) checkOptimismPortal(ctx context.Context) {
 		pausedMetric = 1
 	}
 
-	m.paused.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
+	m.pausedState.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
```

Comment:
The line is using the `optimismPortalAddress` that is correct here.
```go
m.pausedState.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
```

However, the **L192** (that is outside the current diff) is also using the `optimismPortalAddress` which is not correct there: 
```go
m.latestPresignedPauseNonce.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(latestPresignedNonce))
```
The metrics should be the `m.safeAddress` I guess.

Reply:
ahh good catch!